libs/tokens/src/services/fetchTokenList.ts

```diff
@@ -63,13 +70,32 @@ async function _fetchTokenList(source: string, urls: string[]): Promise<ListStat
       continue
     }
 
-    const json = await response.json()
+    try {
+      const json = await response.json()
 
-    return {
-      source,
-      list: await validateTokenList(json),
+      return {
+        source,
+        list: await sanitizeList(json),
+      }
+    } catch (e) {
+      const message = `failed to process list ${url}`
+      console.debug(message, e)
+
+      if (isLast) throw new Error(message)
+
+      continue
```

Comment:
This check didn't exist. If a list was invalid, it would break the loop and not continue with the remaining.